### DISCUSSION TITLE
Is Spacedrive portable across OSes?

### DISCUSSION THREAD

**Discussion by sergeyyurkov1** [spacedriveapp/spacedrive#2503] [General]
If I start tagging my files on Windows but later decide to switch to Linux, will I be able to transfer my tags to the new OS and continue seamlessly, or will I lose the existing tags and need to start from scratch? Thank you.

**Comment by HeavenVolkoff** [comment]
As soon as P2P and Cloud Sync features are available for general use, you will be able to sync all your libraries from one device to another (as well as their attached metadata, such as: tags) and explore them seamlessly no matter from which device you are using
